src/main/java/com/juniorstart/juniorstart/model/Project.java

```diff
@@ -0,0 +1,68 @@
+package com.juniorstart.juniorstart.model;
+
+import com.juniorstart.juniorstart.model.audit.UserDateAudit;
+import jakarta.validation.constraints.NotEmpty;
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Table(name = "projects", uniqueConstraints = {
+        @UniqueConstraint(columnNames = "name")
+})
+@Getter
+@Setter
+@NoArgsConstructor
+@Builder
+@AllArgsConstructor
+public class Project  extends UserDateAudit {
+
+    @Id
+    @Column(name = "project_id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "name", nullable = false)
+    @NotEmpty(message = "Please provide a name")
+    private String name;
+
+    @Column(name = "title", nullable = false)
+    @NotEmpty(message = "Please provide a title")
+    private String title;
+
+    @Column(name = "description", nullable = false)
+    @NotEmpty(message = "Please provide a description")
+    private String description;
+
+    @Column(name = "body", columnDefinition = "TEXT", nullable = false)
+    @NotEmpty(message = "Please provide a body")
+    private String body;
+
+    @Column(name = "number_of_seats", nullable = false)
+    @NotEmpty(message = "Please provide a number of seats")
+    private long numberOfSeats;
+
```

Comment:
Jeżeli nazwa zmiennej jest taka sama jak @Coumn(name = "...") to nie musisz deklarować bo domyślnie on przybiera taką nazwę. Może lepiej jest po prostu zmienne nazwać tak jak potrzebujesz a nie deklarować w @Column